app/javascript/components/supplemental_application/sections/ParkingInformationInputs.js

```diff
@@ -0,0 +1,53 @@
+import React, { useState } from 'react'
+
+import FormGrid from '~/components/molecules/FormGrid'
+import { SelectField, CurrencyField } from '~/utils/form/final_form/Field.js'
+import validate from '~/utils/form/validations'
+
+const monthlyRentFieldName = 'lease.monthly_parking_rent'
+const Yes = 'Yes'
+const No = 'No'
+
+const validateLeaseCurrency = (value) => {
```

Comment:
this is duplicated from the other file. I wonder if it can be moved from to the validations file, or if there's another way to avoid this code duplication

Reply:
Brought this out into a util for both files